admin/modules/user/groups.php

```diff
@@ -964,7 +972,15 @@
 			// Log admin action
 			log_admin_action($usergroup['gid'], htmlspecialchars_uni($mybb->input['title']));
 
-			flash_message($lang->success_group_updated, 'success');
+			$groups = $cache->read('usergroups');
+			$grouptitles = array_column($groups, 'title');
+
+			$message = $lang->success_group_updated;
+			if(in_array($mybb->input['title'], $grouptitles) && count(array_keys($grouptitles, $mybb->input['title'])) > 1) {
```

Comment:
Same here

Reply:
Fixed in 85e640af00f8be38b5ac5e50fcfea3d7850c5727